wordpress-mobile/WordPress-iOS
# Feature/media updates on text mode

This PR allows editing of posts while media uploads are happening.
If media uploads are finished while in text mode the text gets update with the final information.
This functionality is only available in Aztec.

To test:
 - Start a new post.
 - Start media uploads.
 - Switch to html/text mode while uploads are ongoing.
 - Wait to them to be finish and see text being updated.
 - Do the same but switch to html and back to visual mode.
